slackclient/_client.py

```diff
@@ -27,6 +27,12 @@ def __init__(self, token):
         self.token = token
         self.server = Server(self.token, False)
 
+    def get_user_agent(self):
```

Comment:
I feel like `get_user_agent` and `append_user_agent` could be combined. I'm also not sure if `get_user_agent` is necessary as a public method though perhaps @paulhammond has stronger opinions or more knowledge here? What do you think Paul?